packages/integration-sdk-core/src/types/relationship.ts

```diff
@@ -32,7 +32,7 @@ type RelationshipPropertyValue =
   | null
   | RelationshipMapping;
 
-export type TargetFilterKey = string | string[];
+export type TargetFilterKey = string[];
```

Comment:
This is technically a major _type_s change, but not a major _functionality_ change, so I would prefer to release a minor version here rather than a major version.

Reply:
@softwarewright can you confirm that the following two configurations are equivalent?

```ts
  targetFilterKeys: ['<ROOT>']

  targetFilterKeys: [['<ROOT>']]
```

Reply:
@ndowmon `[['<ROOT_ENTITY>']]` is not equivalent to `['<ROOT>']`